x/programs/examples/simulator/cmd/config.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package cmd
+
+import (
+	"github.com/ava-labs/hypersdk/x/programs/runtime"
+)
+
+func newConfig(action *Action, config *Config) (*runtime.Config, error) {
+	return runtime.NewConfigBuilder(action.MaxFee).
+		WithEnableTestingOnlyMode(true).
+		WithBulkMemory(true).
```

Comment:
Don't we always disable this on-chain? Or is it needed for some debugging thing?

Reply:
will add note